patches/esbuild-plugin-babel+0.2.3.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/esbuild-plugin-babel/src/index.js b/node_modules/esbuild-plugin-babel/src/index.js
```

Comment:
Do we still need this if we use babel?

Reply:
We need this is we go with the patch-package implementation. We still need to have that discussion.